rpi-update

```diff
@@ -296,13 +296,24 @@ function noobs_fix {
 
 }
 
+echo " *** Raspberry Pi firmware updater by Hexxeh, enhanced by AndrewS and Dom"
+echo
+echo " *** This script cannot uninstall any firmware automatically, only install different versions onto the system. Use with caution."
+echo " ***  For inexperienced users it is recommended to use their distro-provided packages (e.g. 'raspberrypi-bootloader' or"
```

Comment:
Gee, suddenly a lot of things happening after such a long time. xD Thank you for the feedback, updating!